_core/smart/neon-node.md

```diff
@@ -1,17 +1,17 @@
 ---
-layout: core
-description: "Blockstack smart contracting language"
+layout: smart
+description: "Run a Stacks Testnet Node"
 permalink: /:collection/:path.html
 ---
 # Running a Neon Testnet Node
 {:.no_toc}
 
-"Neon" is phase 1 of the Stacks 2.0 testnet. In Neon, you can run a Stacks node and participate in proof-of-burn mining.
+"Neon" is phase 1 of the Stacks 2.0 testnet. In Neon, you can run a node and connect it to a public network. This guide will walk you through downloading and running your own node in the Neon network.
 
 * TOC
 {:toc}
 
-### Download and build stacks-blockchain
+### Download the `stacks-blockchain` repository
 
 The first step is to ensure that you have Rust and the support software installed.
 
```

Comment:
We should include an explanation for the command prompt in case Rust is already installed. Should we suggest to continue with installation to ensure the latest version is used or not?